chia/plotting/plot_tools.py

```diff
@@ -316,3 +316,40 @@ def find_duplicate_plot_IDs(all_filenames=None) -> None:
         for filename_str in duplicate_filenames:
             log_message += "\t" + filename_str + "\n"
         log.warning(f"{log_message}")
+
+
+def load_plot(
+    filename: Path,
```

Comment:
is filename allowed to be `None`? if so, this should be made `Optional[Path]`. I don't think so though, then I think the check `if filename is None` below should be an `assert filename is not None`

Reply:
You are right. It has been corrected